test/acceptance/admin_test.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Community.Acceptance.AdminTest do
+  use Community.AcceptanceCase
+
+  test "it allows admins to see the jobs" do
+    job = create(:job)
+
+    navigate_to "/admin/jobs"
+
+    assert flash_text =~ "You must log in to see this page"
+
+    navigate_to "/session/new"
```

Comment:
It's not redirecting to the login page. Just for minor security through obscurity it's going to drive you back to root.

Good catch though!